.buildkite/pipeline.yml

```
@@ -13,13 +13,31 @@ steps:
     commands:
       - bundle install
       - bundle exec rake plugin:export
+      - zip -r upm-package.zip upm-package
     artifact_paths:
       - Bugsnag.unitypackage
+      - upm-package.zip
+
     retry:
       automatic:
         - exit_status: "*"
           limit: 1
 
+  - label: "Test UPM Package Import"
+    depends_on: build_unitypackage
+    timeout_in_minutes: 10
+    env:
+      UNITY_VERSION: *2021
+    key: "import-upm-2020"
```

Not sure what the relevance of the key is, but presumably should be 2021?

```suggestion
    key: "import-upm-2021"
```